src/audio/mixin_mixout/mixin_mixout.c

```diff
@@ -448,7 +367,7 @@ static int mixin_copy(struct comp_dev *dev)
 		if (!sink_c->hw_params_configured) {
 			comp_err(dev, "Uninitialized mixout sink buffer!");
 			buffer_release(sink_c);
```

Comment:
@lyakh the buffer_acquite(source) part will be removed in the PR #6836. So that leaves just 2 and I will verify if the ordering if correct in that one